packages/core/src/types.ts

```diff
@@ -2154,11 +2156,31 @@ export interface ActorSystemInfo {
 }
 
 export interface ActorSystem<T extends ActorSystemInfo> {
+  /**
+   * @private
+   */
   _bookId: () => string;
+  /**
+   * @private
+   */
   _register: (sessionId: string, actorRef: AnyActorRef) => string;
+  /**
+   * @private
+   */
   _unregister: (actorRef: AnyActorRef) => void;
+  /**
+   * @private
+   */
   _set: <K extends keyof T['actors']>(key: K, actorRef: T['actors'][K]) => void;
   get: <K extends keyof T['actors']>(key: K) => T['actors'][K] | undefined;
+  inspect: (observer: Observer<InspectionEvent>) => void;
+  _sendInspectionEvent: (event: InspectionEvent) => void;
+  sendTo: (
```

Comment:
I like this method but I wonder about:
- could we use a different order of arguments, having `event` at the end would look visually better when we call this function
- could we make `target` and `source` required?

Reply:
Sure, we can reorder it. It should be an internal-only method too, not meant for public usage.

Reply:
> It should be an internal-only method too, not meant for public usage.

This has crossed my mind but it actually should be available for all custom actors so we can't just `_` it

Reply:
Renamed to `relay(event, source, target)` as it is _relaying_ an event